### PR TITLE
Drop `num_actions` from DiscreteDP

### PR DESCRIPTION
**Pull request by a-parida12**
solves #103

### CODE REVIEW

**Comment by sglyon** [comment]
Thanks for taking this on!

I think my preference would be to just delete the function, but maintain the naming of variables inside our other functions (e.g. still use `num_states, num_actions` instead of `m, n`).

I'd like to get some other opinions on that though... cc @oyamad @cc7768 @mmcky @jstac

**Comment by a-parida12** [comment]
@sglyon I don't fully understand which particular function to be dropped then.

**Comment by sglyon** [comment]
Sorry for not being clear on this one.

You did delete the proper function, but in a few instances you also changed the name of some variables (in the DiscreteDP constructor as well as a few docstrings) away from num_actions to the single letter n.

We should keep the descriptive variable names, but still delete the function as you did. I think that we want to end up with a 1-line change relative to the current master branch that just removes the function definition. We should also check `src/QuantEcon.jl` to make sure `num_actions` is not being exported.

File: src/markov/ddp.jl [hunk 1]
```diff
@@ -577,7 +577,7 @@ end
 
 """
 Return the `Vector` `max_a vals(s, a)`,  where `vals` is represented as a
-`AbstractMatrix` of size `(num_states, num_actions)`.
+`AbstractMatrix` of size `(num_states, num_actions).
```
Comment (sglyon) [inline src/markov/ddp.jl]: I think here we need a closing backtick character after `num_actions)

File: src/markov/ddp.jl [hunk 2]
```diff
@@ -589,7 +589,7 @@ s_wise_max!(vals::AbstractMatrix, out::AbstractVector) = (println("calling this
 
 """
 Populate `out` with  `max_a vals(s, a)`,  where `vals` is represented as a
-`AbstractMatrix` of size `(num_states, num_actions)`.
+`AbstractMatrix` of size `(num_states, num_actions).
```
Comment (sglyon) [inline src/markov/ddp.jl]: Same comment about backtick here

Comment (a-parida12) [reply src/markov/ddp.jl]: yes, my bad. I will improve the formatting.

**Comment by sglyon** [comment]
Thank you

**Pull request merged**